src/main/java/nl/WonderGem/techItEasyWonderGem/service/TelevisionService.java

```diff
@@ -0,0 +1,145 @@
+        this.reposTV = reposTV;
+        this.reposCiMo = reposCiMo;
+        this.reposReCo = reposReCo;
+        this.reposWaBr = reposWaBr;
+    }
+
+
+    public Iterable<TelevisionDto> getAllTelevisions() {
+
+        List<Television> reposTVList = reposTV.findAll();
+        List<TelevisionDto> telDtoList = new ArrayList<>();
+
+        for (Television t :
+                reposTVList) {
+
+            telDtoList.add(TelevisionDto.fromTelevision(t));
+
+        }
+
+        return telDtoList;
+    }
+
+    public TelevisionDto getTelevisionByID(long id) {
+
+        if (Utility.idChecker(id, reposTV)){
+            Television requestedTelevision = reposTV.findById(id).get();
+            return TelevisionDto.fromTelevision(requestedTelevision);
+
+        } else {
+            return null;
+        }
+    }
+
+    public long createTelevision(TelevisionInputDto telInputDto) {
+
+        Television telSaved = reposTV.save(telInputDto.toTelevision());
+
+        return telSaved.getId();
+    }
+
+    public boolean deleteTelevision(Long id) {
+
+        if (Utility.idChecker(id, reposTV)){
+            reposTV.deleteById(id);
+            return true;
+
+        } else {
+
+            return false;
+        }
+
+    }
+
+    public boolean updateTelevision(Long id, TelevisionUpdateInputDto telUpdateInputDto) {
+        if (Utility.idChecker(id, reposTV)){
+            Television tobeUpdated = reposTV.findById(id).get();
+            Television updatedTelevision = telUpdateInputDto.toUpdatedTelevision(tobeUpdated);
+            reposTV.save(updatedTelevision);
+            return true;
+
+        } else {
+
+            return false;
+        }
+
+
+    }
+
+    public boolean assignRemoteControllerToTelevision(Long tvID, Long reCoID) {
+        if (Utility.idChecker(tvID, reposTV) && Utility.idChecker(reCoID, reposReCo)){
+            Television tobeUpdated = reposTV.findById(tvID).get();
+            RemoteController toBeAdded = reposReCo.findById(reCoID).get();
+
+            tobeUpdated.setRemoteController(toBeAdded);
+            reposTV.save(tobeUpdated);
+            return true;
+
+        } else {
+
+            return false;
+        }
+
+    }
+
+    public boolean assignCiModuleToTelevision(Long tvID, Long ciMoID) {
+        if (Utility.idChecker(tvID, reposTV) && Utility.idChecker(ciMoID, reposCiMo)) {
+            Television tobeUpdated = reposTV.findById(tvID).get();
+            CiModule toBeAdded = reposCiMo.findById(ciMoID).get();
+
+            tobeUpdated.addCiModuleToList(toBeAdded);
+            reposTV.save(tobeUpdated);
+            return true;
+
+        } else {
+
+            return false;
+        }
+    }
+
+    public boolean assignWallBracketToTelevision(Long tvID, Long waBrID) {
+        if (Utility.idChecker(tvID, reposTV) && Utility.idChecker(waBrID, reposCiMo)) {
+            Television tobeUpdated = reposTV.findById(tvID).get();
+            WallBracket toBeAdded = reposWaBr.findById(waBrID).get();
+
```

Comment:
Je zou hier nog een check kunnen doen dat je checkt of televisions al een wallbracket heeft met een bepaald id. Je zou daar een custom query voor kunnen schrijven https://docs.spring.io/spring-data/jpa/docs/1.6.0.RELEASE/reference/html/jpa.repositories.html

Als het niet lukt wil ik je daar wel een keer bij helpen.